Board.java

```diff
@@ -1,20 +1,24 @@
+package pacman;
+
 import java.io.BufferedReader;
 import java.io.FileNotFoundException;
 import java.io.FileReader;
+import java.util.Arrays;
 import java.util.Scanner;
 
 public class Board {
 	
-	public Board(){
+	public Board() {
 		this(null);
 	}
 	
-	public Board(String[] rowsArray){
+	public Board( String[] rowsArray ) {
 		this.ghostsNumber = 0;
 		
-		for (Creature c : Creature.values()){
-			c.setLocation(null);
-			c.setNextDirection(null);
+		for (Creature c : MovingCreature.values()){
+			MovingCreature Mc = (MovingCreature) c;

```

Comment:
I don't like the concept of setting values to static members at arbitrary locations. 
If you should initiate these creatures only once, you should do it in its class (MovingCreature). 
If it's part of board initialisation, using static member is wrong.
either way, such init should take place at the MovingCreature class, not here.